content/terraform/v1.13.x/docs/language/meta-arguments.mdx

```diff
@@ -5,6 +5,12 @@ description: Meta-arguments are built-in Terraform arguments that determine how
 
 # Meta-arguments
 
+<Note>
+
+The beta release of Terraform 1.14 lets you trigger automations outside of Terraform with actions. Refer to the [Invoke an action](/terraform/language/v1.14.x/invoke-actions) beta documentation for more information.
```

Comment:
```suggestion
Terraform `v1.14.x (beta)` adds the `action_trigger` meta-argument, which lets you trigger actions that extend Terraform CRUD operations. Refer to the [Invoke an action](/terraform/language/v1.14.x/invoke-actions) beta documentation for more information.
```
Trying a slightly different wording in this one. Use whichever version (or neither) you want. The goal is to use wording that highlitghts the task ("trigger actions that. . . .") instead of the branding (. . . with actions."), if that makes sense.